lib/integrations/base.rb

```diff
@@ -44,14 +56,8 @@ def self.required_settings
       end.compact
     end
 
-    def validate_settings
-      supplied_settings = settings.keys
-      required_settings = self.class.required_settings
-      missing_settings = required_settings - supplied_settings
-
-      unless missing_settings.empty?
-        raise Integrations::Error.new('misconfigured_integration', "Required settings '#{missing_settings.join(", ")}' were not supplied to #{self.class.key}")
```

Comment:
Not raising this anymore because it would interrupt any subsequent notifications from firing off.